drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -193,6 +193,10 @@ enum nrf_wifi_status umac_cmd_init(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
 		umac_cmd_data->disable_beamforming = 1;
 	}
 
+#ifdef CONFIG_NRF_WIFI_QOS_NULLFRAME_PS
+	umac_cmd_data->nullframe_pwrsave = 1;
+#endif /* CONFIG_NRF_WIFI_QOS_NULLFRAME_PS */
+
```

Comment:
as this needs to be sent always, it might be better to do:
```suggestion
	umac_cmd_data->nullframe_pwrsave = IS_ENABLED(CONFIG_NRF_WIFI_QOS_NULLFRAME_PS);

```